modules/data.land/inst/BCI_AGB.R

```diff
@@ -0,0 +1,54 @@
+## Barro Colorado Island AGB timeseries
+library(dplyr)
+indir <- "~/Dropbox/Desktop/Projects/BCI/"
```

Comment:
Vonceivably, but right now this is just documenting how this Input was generated, and would take more work to generalize to something that could be applied to multiple sites. Eventual goal, but for now just want this archived.